src/docs/admin/references/NetworkPropertiesReference.adoc

```diff
@@ -1,107 +1,152 @@
+Configuration files related to network properties for the Scheduler and nodes
+are located respectively in:
 
-- +proactive.communication.protocol+: Represents the communication protocol, i.e. the protocol used to export
- objects on remote JVMs. At this stage, several protocols are supported: RMI(rmi), HTTP(http), IBIS/RMI(ibis),
-  SSH tunneling for RMI/HTTP(rmissh), ProActive Message Routing(pamr). It means that once the JVM starts,
-  nodes and active objects that will be created on this JVM will export themselves using the protocol
-  specified in proactive.communication.protocol property. They will be reachable transparently through
-  the given protocol.
+  - `PROACTIVE_HOME/config/network/server.ini`
+  - `PROACTIVE_HOME/config/network/node.ini`
+
+If you are using ProActive agents, then the configuration file location differs
+depending of the Operating System:
+
+  - On Unix OS, the default location is
+    `/opt/proactive-node/config/network/node.ini`
+  - On Windows OS, the default one is
+    `C:\Program Files (x86)\ProActiveAgent\schedworker\config\network\node.ini`.
+
+==== Common Network Properties
+
+The protocol to use by the Server and the nodes can be configured by setting a
+value to the property `proactive.communication.protocol`. It represents the
+protocol used to export objects on remote JVMs. At this stage, several protocols
+are supported: PNP (pnp), PNP over SSL (pnps), ProActive Message Routing (pamr).
+It means that once the JVM starts, nodes that will be created on this JVM will
+export themselves using the protocol specified in
```

Comment:
'export themselves using the protocol specified' Do you mean that the nodes start connecting immediately?